src/config.ts

```diff
@@ -8,7 +8,13 @@ if(!fs.existsSync(EXPORTED_PATH)) {
 }
 
 export function getApiKey(host) {
-  let data = fs.readFileSync(path.join(__dirname, '../api-keys.json'), 'utf8');
+  let data = fs.readFileSync(path.join(__dirname, '../config.json'), 'utf8');
 
   return JSON.parse(data)[host]
 }
+
+export function getPortNum(port) {
```

Comment:
I don`t like this approach. 

Please see https://github.com/hastic/hastic-server/blob/master/server/src/config.ts

Reply:
changed `export function` to `export const` in `getPortNum` and also in `getApikey`, since both exports were using same approach.

Reply:
@sankerust guess @jonyrock said that you can make more general `getConfigField(field: string, defaultVal?: any)` function
And use `getConfigField('apiKeys')[host]` to get api key for host
And `getConfigField('port')` to get port